src/main/scala/com/codacy/client/bitbucket/v2/service/TeamServices.scala

```diff
@@ -11,6 +11,17 @@ class TeamServices(client: BitbucketClient) {
     client.executePaginated(Request(s"$BaseUrl?role=$role", classOf[Seq[Team]]))
   }
 
+  /**
+    * Gets the public information associated with a team.
+    *
+    * If the team's profile is private, location, website and created_on elements are omitted.
```

Comment:
I would remove this line, as this is details of the Provider that we are not even parsing or returning

Reply:
Sure, its just a copy-paste from Bitbucket docs.